### ISSUE TITLE
Path replacement on Windows doesn't work

### ISSUE THREAD

**Issue by stevygee** [armadillica/flamenco-worker-python#3]
Correct me if I'm wrong, but the Worker should replace path variables like `{job_storage}` with the replacements defined in the Manager, right? Because this is not happening on Windows. I currently can't test on Linux, so I'm not sure if this really is the expected behavior.

```
2017-07-23 15:24:56,761     INFO flamenco_worker.worker.FlamencoWorker Updating task 5974a3a0811917267ea9ea80 with status 'failed' and activity Activity(activity="blender_render.(task_id=5974a3a0811917267ea9ea80, command_idx=0): Invalid settings: filepath '{job_storage}/2017-07-23-152447.664155-stefan.grassberger-test123.flamenco/test123.flamenco.blend' does not exist", current_command_idx=0, task_progress_percentage=0, command_progress_percentage=0)
```
In the Manager config I set `render` to `S:/`, so the expected path would be `S:/2017-07-23-152447.664155-stefan.grassberger-test123.flamenco/test123.flamenco.blend`

**Comment by fsiddi** [comment]
The replacement should happen on the Manager level (based on the worker request). Did you configure the job_storage value correctly in your manager config file?

**Comment by stevygee** [comment]
Ah, that makes sense. I actually changed the platform key in the default Manager config from `windows` to `win32` because I assumed it was a mistake (because the Blender Cloud add-on seems to expect `win32`). But using `windows` the path replacement works!

**Comment by sybrenstuvel** [comment]
This is a known issue with the Blender Cloud add-on, it'll be fixed when we release 1.7.3 (which will happen in a moment).
